### PR TITLE
Fix convertCurrencyWithoutState function multiplication to use Biggystring

### PR DESCRIPTION
**Pull request by amanalili**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android

### CODE REVIEW

File: src/components/themed/WalletList.js [hunk 1]
```diff
@@ -79,10 +79,11 @@ type Props = OwnProps & StateProps & DispatchProps & ThemeProps
 
 class WalletListComponent extends React.PureComponent<Props> {
   sortWalletList(walletList: WalletListItem[]): WalletListItem[] {
-    const getFiatBalance = (wallet: GuiWallet, fullCurrencyCode: string) => {
+    const getFiatBalance = (wallet: GuiWallet, fullCurrencyCode: string): number => {
       const { settings, exchangeRates } = this.props
       const currencyCode = getSortOptionsCurrencyCode(fullCurrencyCode)
-      return calculateWalletFiatBalanceUsingDefaultIsoFiat(wallet, currencyCode, settings, exchangeRates)
+      const fiatBalanceString = calculateWalletFiatBalanceUsingDefaultIsoFiat(wallet, currencyCode, settings, exchangeRates)
+      return parseFloat(fiatBalanceString)
```
Comment (thehobbit85) [inline src/components/themed/WalletList.js:86]: No need to convert back to number.
Instead, when you use this function to compare balances, just use the bns comparison functions: 

https://www.npmjs.com/package/biggystring#comparisons

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See comment

File: src/components/themed/WalletList.js (hunk 1 above)
Comment (amanalili) [reply src/components/themed/WalletList.js:85]: It's not use for comparison but for the Array.sort function so it is needed to have a numerical result.

File: src/selectors/WalletSelectors.js [hunk 2]
```diff
@@ -53,10 +53,10 @@ export const convertCurrency = (state: RootState, fromCurrencyCode: string, toCu
   return convertedAmount
 }
 
-const convertCurrencyWithoutState = (exchangeRates: { [string]: number }, fromCurrencyCode: string, toCurrencyCode: string, amount: number = 1) => {
+const convertCurrencyWithoutState = (exchangeRates: { [string]: number }, fromCurrencyCode: string, toCurrencyCode: string, amount: string = '1'): string => {
```
Comment (peachbits) [inline src/selectors/WalletSelectors.js:56]: The goal is for these functions to only accept strings as arguments. For every argument that passes numbers, like exchangeRates, create a utility function to convert to strings before a function like convertCurrencyWithoutState is called. Then, update convertCurrencyWithoutState to use biggystring.

**Pull request merged**